src/views/playground/Playgrounds/ImageDownsizing.tsx

```diff
@@ -147,9 +147,8 @@ const ImageDownsizing = () => {
             Original image size: {originalAvatarSizes?.width} X {originalAvatarSizes?.height}
           </p>
           <p>Original image filesize: {formatBytes(originalAvatarSizes?.fileSize || 0)}</p>
-          <p>
-            Cropped image size: {avatarSizes.width} X {avatarSizes.height}
-          </p>
+          {/* constant, they are always the same - defined by CANVAS_OPTS_PER_TYPE in cropper.ts  */}
+          <p>Cropped image size: 256 X 256</p>
```

Comment:
I think the point here is to make sure (test) that they actually are of the specified size. Can we just read that from the image?

Reply:
I think we can get that easily with `naturalHeight` & `naturalWidth`?